Library/Homebrew/software_spec.rb

```diff
@@ -208,6 +208,8 @@ def recursive_requirements
   end
 
   def patch(strip = :p1, src = nil, &block)
+    gpatch_dep = gpatch_dep_if_needed(:build)
```

Comment:
```suggestion
    gpatch_dep = DependencyCollector.gpatch_dep_if_needed(:build)
```
`gpatch_dep_if_needed` is a class method of `DependencyCollector`.

Reply:
```suggestion
    gpatch_dep = DependencyCollector.gpatch_dep_if_needed([:build])
```
`gpatch_dep_if_needed` expects an Array of symbols.